Changes

```diff
@@ -218,6 +218,10 @@ _______________
 - #13326: Implement Unix.O_APPEND on windows.
   (Romain Beauxis, review by Miod Vallat, Gabriel Scherer and Antonin Décimo)
 
+- #13376: Allow Dynlink.loadfile_private to load bytecode libraries with
```

Comment:
Mark this as a breaking change (`*`), just in case someone was relying on the old behavior for additional security?

Reply:
Done